sbin/cubeit

```diff
@@ -376,6 +376,148 @@ install_summary()
 
 }
 
+#
+# Copy the installer 'static' parts, that is scripts, dirs,
+# files, libs...
+#
+# Configs are not static and so are dealt with as content
+# and copied later, see copy_installer_data().
+#
+copy_installer()
+{
+	local destdir="$1"
+
+	debugmsg ${DEBUG_INFO} "Copying installer to install media"
+	recursive_mkdir ${destdir}${INSTALLER_TARGET_LIB_DIR} || return 1
+
+	cp -r ${LIBDIR}/* ${destdir}${INSTALLER_TARGET_LIB_DIR}
```

Comment:
A few questions .. why would this ever fail ? and if it does fail, the error message isn't going to tell us why. So why not add a -f to that copy, and test for the directories before doing the copy ? That way we could have a better error message and avoid a failure case.